app/assets/javascripts/discourse/app/components/group-manage-save-button.js

```diff
@@ -38,53 +41,61 @@ export default Component.extend({
     }
   },
 
-  actions: {
-    save() {
-      if (this.beforeSave) {
-        this.beforeSave();
-      }
+  @action
+  setUpdateExistingUsers(value) {
+    this.updateExistingUsers = value;
+  },
 
-      this.set("saving", true);
-      const group = this.model;
+  @action
+  save() {
+    if (this.beforeSave) {
+      this.beforeSave();
+    }
 
-      popupAutomaticMembershipAlert(
-        group.id,
-        group.automatic_membership_email_domains
-      );
+    this.set("saving", true);
+    const group = this.model;
 
-      const opts = {};
-      if (this.updateExistingUsers !== null) {
-        opts.update_existing_users = this.updateExistingUsers;
-      }
+    popupAutomaticMembershipAlert(
+      group.id,
+      group.automatic_membership_email_domains
+    );
 
-      return group
-        .save(opts)
-        .then(() => {
-          this.setProperties({
-            saved: true,
-            updateExistingUsers: null,
-          });
+    const opts = {};
+    if (this.updateExistingUsers !== null) {
+      opts.update_existing_users = this.updateExistingUsers;
+    }
 
-          if (this.afterSave) {
-            this.afterSave();
-          }
-        })
-        .catch((error) => {
-          const json = error.jqXHR.responseJSON;
-          if (error.jqXHR.status === 422 && json.user_count) {
-            const controller = showModal("group-default-notifications", {
-              model: { count: json.user_count },
-            });
+    return group
+      .save(opts)
+      .then(() => {
+        this.setProperties({
+          saved: true,
+          updateExistingUsers: null,
+        });
+
+        if (this.afterSave) {
+          this.afterSave();
+        }
+      })
+      .catch((error) => {
+        const json = error.jqXHR.responseJSON;
+        if (error.jqXHR.status === 422 && json.user_count) {
+          this.editGroupNotifications(json);
+        } else {
+          popupAjaxError(error);
+        }
+      })
+      .finally(() => this.set("saving", false));
```

Comment:
`saving` doesn't seem to be used anywhere?

Reply:
We use this for the `savingText`

https://github.com/discourse/discourse/pull/22521/files#diff-2bf398903ccc368349114cd35a8d6d36fced1a77f3d3ff487b5641dea72e0323R18-R21